system-tests/docker-test-env/docker-compose.yml

```diff
@@ -1,17 +1,18 @@
 version: '3'
 services:
-
-  httpd-01:
-    image: httpd:2.4
+  origin:
+    image: nginx:stable-alpine
     ports:
-      - "8081:80"
+      - "9091:8080"
```

Comment:
I would rather start the nginx listener in the same ports we are going to expose externally.

Reply:
Hi @dvlato 

I'd like to keep all http services behind the same port (8080). It is easier to configure all of styx, toxi-proxy, and nginx etc when the port is always the same. The ports won't conflict inside docker network: each service will have its own host name.

Therefore the ports are exposed only for the debugging purposes, and on the host hey need to be unique.

Therefore I'd keep it as it is.